### ISSUE TITLE
Unnecessary using (S1128) false positive when used only in comments

### ISSUE THREAD

**Issue by DuAell** [SonarSource/sonar-dotnet#2737]
### Description

When the 'using' is not used in code, but only in comments, S1128 pops up. It shouldn't do because if the using is removed, code doesn't compile anymore

### Repro steps

1. Create an interface with a 'using'
```
using Framework.Exceptions;
```

2. Declare a method
```
/// <summary>
/// Start service. If service is already running, nothing happens.
/// </summary>
/// <exception cref="WindowsServiceException">thrown if service cannot be started.</exception>
void Start();
```
### Expected behavior

S1128 shouldn't be detected as there is no error there

### Actual behavior

A code smell is detected

### Related information

* SonarC# Version : 7.17 (build 9346)
* If running through the Scanner for MSBuild, its version : 4.7.1.2311

**Comment by Corniel** [comment]
This is a duplicate of #2694.
